pkg/printers/internalversion/describe.go

```diff
@@ -3482,6 +3485,38 @@ func describeStorageClass(sc *storage.StorageClass, events *api.EventList) (stri
 	})
 }
 
+func printAllowedTopologies(w PrefixWriter, topologies []api.TopologySelectorTerm) {
```

Comment:
Can you paste an example of the output?

Reply:
![out](https://user-images.githubusercontent.com/12265352/44245103-b7fcbf00-a209-11e8-82ae-1d18c2b0e1af.JPG)

Similar to that of node affinity